### PR TITLE
added Logout page

### PR DESCRIPTION
**Pull request by Ankur9669**
image

### CODE REVIEW

**Review by eakansh125 — APPROVED** [review]
Could not find any syntactic issues due to very less elements on screen.
The code looks clean as well. Can be merged from my side!

File: logout.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    <link rel="stylesheet" href="signup.css" />
+</head>
+
+<body>
+    <!--  ******************************** Navbar Start ********************************  -->
+    <nav class="navbar">
+        <div aria-label="menu-icon">
+            <img alt="hamburger-icon" src="images/hamburger.svg" class="hamburger-icon" id="hamburger-icon" />
+        </div>
+        <div class="logo-container" aria-label="fitbit-logo">
```
Comment (parvez-rashid) [inline logout.html:25]: Great that you've used `aria-` property for semantic HTML!

File: logout.html [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+            <div class="logo">
+                <img class="logo-image" alt="logo-image" src="images/neatUI.svg" />
+                <span class="logo-text">FiT</span>
+                BiT
+            </div>
+        </div>
+        <div class="nav-items-container-end">
+            <div class="login-search-bar-container">
+                <a href="login.html" class="button button-primary">Login</a>
+                <div class="input-box" aria-label="search-box">
```
Comment (parvez-rashid) [inline logout.html:35]: Great that you've used `aria-` property for semantic HTML!

**Review by parvez-rashid — APPROVED** [review]
The UI looks great. Amazing work overall on the code, can be merged.

File: logout.html (hunk 1 above)
Comment (Ankur9669) [reply logout.html:25]: Thanks😂

File: signup.css [hunk 3]
```diff
@@ -114,6 +114,17 @@
...
+    min-height: 80vh;
+}
+.logout-btn-container{
+    width: 100%;
+    display: flex;
+    justify-content: center;
+}
 @media only screen and (max-width: 1024px){
     .authentication{
         grid-template-columns: 100%;
```
Comment (Sahil25061999) [inline signup.css:130]: You can use em here. Rest of the code looks good. Just a suggestion for logout page there are too much primary button, if possible also use a secondary button, not  necessary though.

**Review by Sahil25061999 — APPROVED** [review]

**Pull request merged**